build-native-with-native.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e # exit on error
+set -x # echo on
+
+if [[ "$MSYSTEM" != "MINGWARM64" ]]; then
+  echo "This script must be run in the MSYS2 MINGWARM64 shell"
+  exit 1
+fi
+
+ROOT_DIR=`dirname ${BASH_SOURCE[0]}`
+ROOT_DIR=`realpath $ROOT_DIR`
+
+export FLAVOR=NATIVE_WITH_NATIVE
+export CLEAN_BUILD=1
+
+function build_package () {
+  PACKAGE=$1
+  REPOSITORY=${2:-MINGW}
+  echo "::group::Build $PACKAGE"
+    pushd ../$REPOSITORY-packages/$PACKAGE
+      INSTALL_PACKAGE=1 \
+        $ROOT_DIR/.github/scripts/build-package.sh $REPOSITORY
+    popd
+  echo "::endgroup::"
+}
+
+.github/scripts/setup-repository.sh
+.github/scripts/setup-mingwarm64.sh
+
+.github/scripts/clean-native.sh
+.github/scripts/install-dependencies.sh
+.github/scripts/enable-ccache.sh
+
+#build_package mingw-w64-libtool
```

Comment:
Build of those packages will be provided in the next interation removing some additional workarounds.